dask/array/slicing.py

```diff
@@ -585,23 +586,23 @@ def take(outname, inname, chunks, index, itemsize, axis=0):
 
     Mimics ``np.take``
 
+    >>> from pprint import pprint
     >>> chunks, dsk = take('y', 'x', [(20, 20, 20, 20)], [5, 1, 47, 3], 8, axis=0)
     >>> chunks
     ((2, 1, 1),)
-    >>> dsk  # doctest: +SKIP
-    {('y', 0): (getitem, (np.concatenate, [(getitem, ('x', 0), ([1, 3, 5],)),
-                                           (getitem, ('x', 2), ([7],))],
-                                          0),
-                         (2, 0, 4, 1))}
+    >>> pprint(dsk)
+    {('y', 0): (<built-in function getitem>, ('x', 0), (array([5, 1]),)),
+     ('y', 1): (<built-in function getitem>, ('x', 2), (array([7]),)),
+     ('y', 2): (<built-in function getitem>, ('x', 0), (array([3]),))}
```

Comment:
The output seems to have changed a lot from what's currently in the doctest. I could definitely use a second pair of eyes, especially regarding the dict ordering.

Same applies for the changes a few lines further down.

Reply:
Yeah if you look at the git blame it's not surprising this has changed so dramatically. It looks like that skipped output is 7 years old. https://github.com/dask/dask/blame/main/dask/array/slicing.py#L588-L595